README.md

```diff
@@ -1,16 +1,18 @@
 [comment]: # ( Copyright Contributors to the Open Cluster Management project )
 
-# IAM Policy Controller [![KinD tests](https://github.com/open-cluster-management/iam-policy-controller/actions/workflows/kind.yml/badge.svg?branch=main&event=push)](https://github.com/open-cluster-management/iam-policy-controller/actions/workflows/kind.yml)
+# IAM Policy Controller
+
+[![Build](https://img.shields.io/badge/build-Prow-informational)](https://prow.ci.openshift.org/?repo=open-cluster-management%2Fcert-policy-controller) [![KinD tests](https://github.com/open-cluster-management/iam-policy-controller/actions/workflows/kind.yml/badge.svg?branch=main&event=push)](https://github.com/open-cluster-management/iam-policy-controller/actions/workflows/kind.yml) [![License](https://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
 
 ## Description
-IAM policy controller watches cluster administrator role and IAM role binding created and used within a Kubernetes cluster. It detects the cluster administrator role and IAM role binding violations and report it. The controller shows whether or not a given `IamPolicy` is compliant or noncompliant.
 
-## Usage
-The controller can be run as a stand-alone program or as an integrated part of governing risk with Red Hat Advanced Cluster Management for Kubernetes.
+The IAM policy controller is a controller that watches `IamPolicies` created to monitor IAM cluster role bindings used within a Kubernetes cluster. It detects the number of cluster role bindings and reports whether or not a given `IamPolicy` is compliant or noncompliant.
```

Comment:
Can you add `The controller can be run as a stand-alone program or as an integrated part of governing risk with the Open Cluster Management project.`? The key is to mention this is part of open cluster management project.